pom.xml

```diff
@@ -48,17 +48,17 @@ under the License.
     </modules>
 
     <properties>
-        <flink.version>1.16.0</flink.version>
+        <flink.version>1.17.0</flink.version>
 
-        <junit5.version>5.8.1</junit5.version>
+        <junit5.version>5.9.1</junit5.version>
         <assertj.version>3.23.1</assertj.version>
-        <archunit.version>0.22.0</archunit.version>
+        <archunit.version>1.0.0</archunit.version>
         <testcontainers.version>1.17.2</testcontainers.version>
-        <mockito.version>2.21.0</mockito.version>
+        <mockito.version>3.4.6</mockito.version>
         <hamcrest.version>1.3</hamcrest.version>
 
         <japicmp.skip>false</japicmp.skip>
-        <japicmp.referenceVersion>1.16.0</japicmp.referenceVersion>
+        <japicmp.referenceVersion>1.17.0</japicmp.referenceVersion>
```

Comment:
@zentol Is it correct that we also need to update this when introducing support for a new Flink minor version?

Reply:
No. This should point to the last released version of the connector.

Reply:
So it should be 3.0.0 for this connector